src/core/connection.c

```diff
@@ -7062,7 +7053,14 @@ QuicConnParamSet(
             break;
         }
 
-        Status = QuicConnAddLocalAddress(Connection, (QUIC_ADDR*)Buffer);
+        QUIC_ADDR* LocalAddress = (QUIC_ADDR*)Buffer;
+
+        if (!QuicAddrIsValid(LocalAddress)) {
```

Comment:
Personally, I'd have combined this with the other invalid parameter check (on the same line) as the one above. Looks a bit cleaner IMO.

Reply:
Now, all the checks are done on the same line.